frontend/public/components/monitoring/receiver-forms/routing-labels-editor.tsx

```diff
@@ -84,15 +89,23 @@ export const RoutingLabelEditor = ({ formValues, dispatchFormChange, isDefaultRe
 
   return (
     <div data-test-id="receiver-routing-labels-editor" className="form-group">
-      <SectionHeading text="Routing Labels" required={!isDefaultReceiver} />
+      <SectionHeading
+        text={t('routing-labels-editor~Routing labels')}
+        required={!isDefaultReceiver}
+      />
       <p className="co-help-text">
-        Firing alerts with labels that match all of these selectors will be sent to this receiver.
-        Label values can be matched exactly or with a &nbsp;
-        <ExternalLink href="https://github.com/google/re2/wiki/Syntax" text="regular expression" />.
+        {t(
+          'routing-labels-editor~Firing alerts with labels that match all of these selectors will be sent to this receiver. Label values can be matched exactly or with a ',
+        )}
+        <ExternalLink
+          href="https://github.com/google/re2/wiki/Syntax"
+          text={t('routing-labels-editor~regular expression')}
+        />
+        {t('routing-labels-editor~.')}
```

Comment:
Hm, I'm not sure the best way to deal with this. It really should be one message for translation. We might need to use the `Trans` component here.